idc/views.py

```diff
@@ -214,7 +214,6 @@ def quota_page(request):
 
 
 # Data exploration and cohort creation page
-@login_required
 def explore_data_page(request):
```

Comment:
Have to leave this in until we have the go-ahead to launch, sadly. I took it out locally and it seems to work good! Thanks. :)